backend/src/main/java/de/neuefische/backend/api/dto/ApiResponseAddressDetails.java

```diff
@@ -0,0 +1,11 @@
+package de.neuefische.backend.api.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public record ApiResponseAddressDetails(
+        @JsonProperty("plz")
+        String postalCode,
+        @JsonProperty("name")
```

Comment:
I see in the declaration "String city" but in the annotation only "name". I would recommend also "city" or "cityName" in the annotation! (...?)

Reply:
The 'name' comes from the external API, where it actually stores the names of cities. That's why I use city and the @JsonProperty("name") annotation.